mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulGameScreenTest.kt

```diff
@@ -797,6 +803,28 @@ class StatefulGameScreenTest {
     robot.onNodeWithLocalizedText { gameListening }.performClick()
     robot.onNodeWithLocalizedContentDescription { gameMicOffContentDescription }.assertExists()
   }
+
+  @Test
+  fun given_enabled_volume_button_when_clicked_then_disable() = runTest {
+    val user = mockk<AuthenticatedUser>()
+    every { user.uid } returns "userId1"
+    every { user.name } returns "userName1"
+
+    val env =
+        rule.setContentWithTestEnvironment {
```

Comment:
You should use the recently added `rule.setContentWithAuthenticatedTestEnvironment` instead, which would not require you to mock the user.